src/parameters/parameters_ode_solver.cpp

```diff
@@ -7,6 +7,26 @@ void ODESolverParam::declare_parameters (dealii::ParameterHandler &prm)
 {
     prm.enter_subsection("ODE solver");
     {
+        prm.declare_entry("atol", "0.001",
+                          dealii::Patterns::Double(),
+                          "Absolute Tolerance");
```

Comment:
Could you add a little more detail to these comments? Maybe add "For automatic step size controller". 

It would be nice to also have these in their own subsection, but that's up to preference! ( see the line with ` prm.enter_subsection("rrk root solver");` for an example).